tools/nimgrep.nim

```diff
@@ -8,15 +8,15 @@
 #
 
 import
-  os, strutils, parseopt, pegs, re, terminal
+  os, strutils, parseopt, pegs, re, terminal, osproc, tables, algorithm, times
 
 const
-  Version = "1.5"
+  Version = "1.6"
   Usage = "nimgrep - Nim Grep Utility Version " & Version & """
 
   (c) 2012 Andreas Rumpf
 Usage:
-  nimgrep [options] [pattern] [replacement] (file/directory)*
+  nimgrep [options] pattern [replacement] (file/directory)*
```

Comment:
note that pattern would still be optional in some cases eg `nimgrep -v` or `nimgrep -h`

Reply:
those cases are too marginal and obvious to mention them in main Usage. In new expanded version of --help Usage is split to 3 basic variants (search, replace, filenames). Hopefully what's required and what's not is clearer now.

Reply:
it is, thanks!